frontend/online-shop/src/components/ProductList.tsx

```diff
@@ -0,0 +1,69 @@
+import ProductItem from "./ProductItem";
+import { productListStyle } from "../styles/ProductListStyle";
+
+var data = [
```

Comment:
Use `const` instead of `var`. Here's why: https://janakachathuranga.medium.com/why-we-use-const-and-avoid-var-3528ee550fb4. Also you should declare a `Product` model and give this data a type like `const products: Product[] = ...`

Reply:
Fixed